frontend/packages/metal3-plugin/src/components/host-menu-actions.tsx

```diff
@@ -60,10 +60,12 @@ export const RemoveNodeMaintanance = (
 
 export const menuActions = [SetNodeMaintanance, RemoveNodeMaintanance];
 
+type ExtraResources = { machines: MachineKind[]; nodeMaintenances: K8sResourceKind[] };
```

Comment:
Fixing types to work with [improved `KebabActionsCreator`](https://github.com/openshift/console/pull/2106/files#diff-45aa3dfc05205bd3d1b3da31bd999e6dR101).